compiler/tfldump/src/OpPrinter.cpp

```diff
@@ -592,6 +592,20 @@ class WhilePrinter : public OpPrinter
   }
 };
 
+class UniquePrinter : public OpPrinter
+{
+public:
+  void options(const circle::Operator *op, std::ostream &os) const override
```

Comment:
`circle` -> `tflite`

Reply:
Thank you for spotting, I have corrected the line.